### ISSUE TITLE
colloquialSinceNow should fallback to timeComponentsSince now when unitStyle is set to abbreviated form

### ISSUE THREAD

**Issue by daviskoh** [malcommac/SwiftDate#463]
Version from Podfile: `pod 'SwiftDate', '~> 4.1.6'`

Unless I'm misunderstanding the docs:

![image](https://user-images.githubusercontent.com/5395403/28739524-988cb446-73c9-11e7-98be-3d8c7cf085c2.png)

This:

```
var date = DateInRegion()
date = date - 1.hour

let sinceNow = try! 
date.colloquialSinceNow(style: .abbreviated)

print(sinceNow.colloquial)
```

Should print something along the lines of:

```
1h
```
or
```
1h ago
```

But rather it is printing:

```
one hour ago
```

Which seems to be style `spellOut`. 

I also tried all of the other `UnitsStyle`s and they produce the same results. Am I using the function correctly? Is this a known issue?

**Comment by daviskoh** [comment]
It seems to be that [SwiftDate.strings](https://github.com/malcommac/SwiftDate/blob/master/Sources/SwiftDate/SwiftDate.bundle/en-US.lproj/SwiftDate.strings#L30) files do not support abbreviation. Perhaps we can add a new `localizedKey`. For various `DateComponentsFormatter.UnitsStyle`s though this would obviously increase the sizes of `SwiftDate.strings` files exponentially...

**Comment by malcommac** [comment]
Hi,
as you said colloquial ignore the `unitStyle` parameter. You should fallback to `timeComponents` function.

```swift
let opts = ComponentsFormatterOptions(style: .abbreviated, zero: .dropLeading)
let string = try! date.timeComponentsSinceNow(options: opts) // 1h 
```

I think we should avoid adding other localizations entries to describe what, in fact, is time component. However the best way is to fallback automatically to `timeComponents` function when `unitStyle` is set to an abbreviated form.

I'll post this fix in 4.1.8 release.
Thank you for your report @daviskoh
